server/src/termdb.gdc.js

```diff
@@ -1201,7 +1181,7 @@ async function caseCacheIsStale(ds) {
 
 	let value
 	try {
-		value = await getApiStatus(ds)
+		value = await ds.getStatus().data_release_version
```

Comment:
@siosonel @aacic coding err that should be ds.preInit.getStatus()
but how come you guys tested it to work??